decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -600,6 +600,12 @@ def generate_localized_title
         some_extra_data: "1"
       }
     end
+
+    trait :comment_notification do
```

Comment:
This is only used in one spec, can we move it there?

Reply:
done 👌